power_grid_model_c/power_grid_model/include/power_grid_model/optimizer/tap_position_optimizer.hpp

```diff
@@ -6,10 +6,14 @@
 
 #include "base_optimizer.hpp"
 
+#include "../all_components.hpp"
 #include "../auxiliary/dataset.hpp"
 #include "../common/enum.hpp"
+#include "../common/exception.hpp"
+#include "../main_core/state_queries.hpp"
 
 #include <boost/graph/compressed_sparse_row_graph.hpp>
+#include <boost/graph/dijkstra_shortest_paths.hpp>
```

Comment:
nitpick: whiteline between 3rd party libs and STL libs

```suggestion
#include <boost/graph/dijkstra_shortest_paths.hpp>

```